ui/src/components/search/Search.js

```diff
@@ -121,67 +202,63 @@ const Search = ({
-                        },
+                    },
+                    {
+                      content: {
+                        fieldName: 'type',
+                        displayName: 'Model Type',
                       },
-                      {
-                        content: {
-                          field: 'has_matched_models',
-                          displayName: (
-                            <Row justifyContent="space-between">
-                              Has Multiple Models
-                              <MultipleModelsTooltip
-                                isFacet={true}
-                                width={state.panelSize - nonSearchableFacetTooltipPadding}
-                              />
-                            </Row>
-                          ),
-                        },
+                    },
+                    {
+                      content: {
+                        fieldName: 'has_matched_models',
+                        displayName: (
+                          <Row justifyContent="space-between">
+                            Has Multiple Models
+                            <MultipleModelsTooltip
+                              isFacet={true}
+                              width={tableState.panelSize - nonSearchableFacetTooltipPadding}
+                            />
+                          </Row>
+                        ),
                       },
-                      {
-                        content: {
-                          field: 'molecular_characterizations',
-                          displayName: (
-                            <Row justifyContent="space-between">
-                              Available Molecular Characterizations
-                              <MolecularCharacterizationsTooltip
-                                isFacet={true}
-                                width={state.panelSize - facetTooltipPadding}
-                              />
-                            </Row>
-                          ),
-                        },
+                    },
+                    {
+                      content: {
+                        fieldName: 'molecular_characterizations',
+                        displayName: (
+                          <Row justifyContent="space-between">
+                            Available Molecular Characterizations
+                            <MolecularCharacterizationsTooltip
+                              isFacet={true}
+                              width={tableState.panelSize - facetTooltipPadding}
+                            />
+                          </Row>
+                        ),
                       },
-                    ]}
+                    },
+                  ]}
+                />
+                {/* TODO: Review componentProps
+                <Aggregations
+                    componentProps={{
+                      getTermAggProps: () => ({ maxTerms: 4 }),
+                    }}
```

Comment:
TODO is just this one prop, I don't know how to integrate this or if it's needed

Reply:
From Slack:
this is still available on the new Aggregation component and I've added it back in, part of the refresh bug commit
https://github.com/nci-hcmi-catalog/portal/commit/cc0b128dafc354d11b0dc1e145fbea31c7c77190

If you look at `arranger/modules/components/src/aggregations/Aggregations` it still accepts componentProps in AggregationListDisplay
https://github.com/overture-stack/arranger/blob/main/modules/components/src/aggregations/Aggregations.jsx#L25